### PR TITLE
[master < T0900-MG] Add mgp package

### PR DESCRIPTION
**Pull request by jbajic**
[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Update [changelog](https://docs.memgraph.com/memgraph/changelog)

### CODE REVIEW

File: release/mgp/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "mgp"
+version = "1.0.0"
+description = "Memgraph's module for developing MAGE modules. Used only for type hinting!"
+authors = [
+    "MasterMedo <mislav.vuletic@gmail.com>",
```
Comment (42jeremy) [inline release/mgp/pyproject.toml:6]: Is it wanted?

**Review by 42jeremy — APPROVED** [review]

File: release/mgp/pyproject.toml (hunk 1 above)
Comment (jbajic) [reply release/mgp/pyproject.toml:6]: I have no preference but I would leave people who have made some contributions, unless we want otherwise

Comment (42jeremy) [reply release/mgp/pyproject.toml:6]: Me neither, it was simply to check it was not a "wrong" copy paste :)

File: release/mgp/_mgp.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+def type_relationship():
+    pass
+
+
+def type_path():
+    pass
+
+
+class _MODULE:
+    def add_read_procedure(wrapper):
```
[the remark this answers is not in the document]
Comment (jbajic) [reply release/mgp/_mgp.py:71]: You are right it is missing, as well `add_function` and `add_procedure`

**Pull request merged**